docs/upgrade-to-1.0.md

```diff
@@ -42,6 +42,19 @@ _*backups created prior to v0.10 stored snapshot metadata in the `status.volumeB
     velero backup-location get
     ```
 
+1. Fetch your bucket credentials and make them accessible to Velero
+
+    ```bash
+    # AWS
+    AWS_SHARED_CREDENTIALS_FILE=$(kubectl get secrets/<YOUR_SECRET_NAME> -n velero -o jsonpath="{.data.cloud}" | base64 -d)
+
+    #GCP
+    GOOGLE_APPLICATION_CREDENTIALS=$(kubectl get secrets/<YOUR_SECRET_NAME> -n velero -o jsonpath="{.data.cloud}" | base64 -d)
+
+    # Azure
+    source $(kubectl get secrets/<YOUR_SECRET_NAME> -n velero -o jsonpath="{.data.cloud}" | base64 -d)
```

Comment:
in the old (pre-1.0) way of providing Azure creds, the secret has a bunch of keys, each of which corresponds to an env var - so we'd need to read each of them and then set the appropriate local env var

Reply:
I don't think we need to do them one at a time, but `source` is the incorrect command.

See this example where I cat a file that's just a list of env var key/value pairs (like the pre-1.0 Azure creds file that's given to `fromEnv`)

```bash
x1c in /home/nrb/go/src/github.com/heptio/velero (git) document-migration-secret
% cat test.env
ONE=one
TWO=two

x1c in /home/nrb/go/src/github.com/heptio/velero (git) document-migration-secret
% source $(cat test.env)
source: no such file or directory: ONE=one

x1c in /home/nrb/go/src/github.com/heptio/velero (git) document-migration-secret
% export $(cat test.env)

x1c in /home/nrb/go/src/github.com/heptio/velero (git) document-migration-secret
% echo $ONE
one

x1c in /home/nrb/go/src/github.com/heptio/velero (git) document-migration-secret
% echo $TWO
two
```